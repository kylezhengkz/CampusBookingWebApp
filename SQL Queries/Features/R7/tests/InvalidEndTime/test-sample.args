{
    "args": [
        "00000000-0000-0000-0000-000000000004",
        "00000000-0000-0000-0000-000000000003",
        "2028-01-17 08:00:05",
        "2028-01-18 23:30:19",
        3
    ],
    "kwargs": {}
}   