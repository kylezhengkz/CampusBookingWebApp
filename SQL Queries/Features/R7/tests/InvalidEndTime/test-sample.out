Booking end time must be earlier than 11:00 PM EST